### PR TITLE
NIFI-5389 Added additional documentation for JsonQueryElasticsearch.

### PR DESCRIPTION
**Pull request by MikeThomsen**
Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [ ] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/resources/docs/org.apache.nifi.processors.elasticsearch.JsonQueryElasticsearch/additionalDetails.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    the latter case, the name of the attribute (Expression Language is supported here) must be provided in the Query Attribute
+    property.</p>
+    <p>Additionally, search results and aggregation results can be split up into multiple flowfiles. Aggregation results
+    will only be split at the top level because nested aggregations lose their context (and thus lose their value) if
+    separated from their parent aggregation. The following is an example query that would be accepted:</p>
+    <pre>
+        {
+            "query": {
+                "match": {
+                    "restaurant.keyword": "Local Pizzaz FTW Inc"
```
Comment (mattyb149) [inline nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/resources/docs/org.apache.nifi.processors.elasticsearch.JsonQueryElasticsearch/additionalDetails.html]: Is this example copy-pasted from somewhere (Elastic, Kibana, etc.)? Just want to make sure we can use it as-is (sometimes they're available under Creative Commons or something), or just to be safe, might want to make our own...

Comment (MikeThomsen) [reply nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/resources/docs/org.apache.nifi.processors.elasticsearch.JsonQueryElasticsearch/additionalDetails.html]: Nope, it's all original.

Comment (mattyb149) [reply nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/resources/docs/org.apache.nifi.processors.elasticsearch.JsonQueryElasticsearch/additionalDetails.html]: Nice, thanks! Will merge shortly

**Comment by mattyb149** [comment]
+1 LGTM, verified the additional details page is present and displayed as intended. Thanks for the documentation improvement! Merging to master

**Pull request closed without merging**